### PR TITLE
Ruby on Ice blog 

### PR DESCRIPTION
**Pull request by mlan080**
This is my first draft of the Ruby on Ice conference post. 

When I ran `hugo server` I got the following: Error: Error building site: "/Users/mandylan/fromatob/blog/content/post/2019-03-06-sled-like-no-one's-watching.md:2:1": failed to unmarshal YAML: yaml: mapping values are not allowed in this context

### CODE REVIEW

File: content/post/2019-03-06-sled-like-no-one's-watching.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Sled like no one's watching: Ruby on Ice Conference 2019
```
Comment (w0st) [inline content/post/2019-03-06-sled-like-no-one's-watching.md]: YAML parser trying to interpret the `:` in the title as a separator.

You can just change it to: `title: "Sled like no one's watching: Ruby on Ice Conference 2019"`
It should work :+1:

File: content/post/2019-03-06-sled-like-no-one's-watching.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+![Example image](/resources/_gen/images/view.png)
+As we entered the conference, we were greeted by a rustic, lecture style hall of 150 people. It was an intimate gathering where it was comfortable to start conversations and share knowledge - both technical and non-technical
+
+Eileen Uchitelle from Github kicked off the opening keynote with their experience on forking rails and how they played catch up in upgrading from Rails 3.2 to 5.2 She outlined challenges in dealing with 3400 errors of incompatible dependencies, outdated gems and security backports. It was inspiring to see her 1 woman team fronting this change within Github.
+
+## Extreme fun
+
+Question: What do programmers do when they're not coding? Extreme sport sledding of course.
+
+Equipped with a wooden sled, crash helmet and no instructions on how to stop,  we slid down at highs of 25mph with a high risk of dropping off the side with  a 20 degree slope with trees.
```
Comment (w0st) [inline content/post/2019-03-06-sled-like-no-one's-watching.md]: You placed two spaces before "we slid down" and "a 20 degree". I think you can reduce it to one.

File: content/post/2019-03-06-sled-like-no-one's-watching.md (hunk 1 above)
Comment (mlan080) [reply content/post/2019-03-06-sled-like-no-one's-watching.md]: Thanks for that @w0st it was really helpful :)

File: content/post/2019-03-06-sled-like-no-one's-watching.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Equipped with a wooden sled, crash helmet and no instructions on how to stop,  we slid down at highs of 25mph with a high risk of dropping off the side with a 20 degree slope with trees.
+
+![Example image](/slopes.jpg)
+
+Perhaps they thought engineers could manage anything by themselves.
+It was a refreshing exprience with the splash of ice down my top after the fifth face plant. It was also flattering to hear this from my fellow rubyonicers: 'Thank you for choosing to snowball down the hill instead of crashing into us!'
+
+### Extreme validations
+
+We rolled straight into learning mode again with Tobias' validation talk which was applicable to our application and probably for all rails applications. Validations can be different based on the context so do validations they really belong into models? Tobias showed some alternetive solutions and gave us a little excursion into Elixir land.
```
Comment (lfritz) [inline content/post/2019-03-06-sled-like-no-one's-watching.md]: Maybe add Tobias's last name here?

File: content/post/2019-03-06-sled-like-no-one's-watching.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Sled like no one's watching: Ruby on Ice Conference 2019"
+date: 2019-03-06
+author: Mandy Lan Andreas Soiron
```
Comment (lfritz) [inline content/post/2019-03-06-sled-like-no-one's-watching.md]: At least when I ran it locally, this created a link to `https://github.com/`. I guess you wanted to have an Instagram link here? Maybe you can ask Lucas for help; he's in charge of the technical side of the blog.

**Pull request merged**